streams/core.py

```diff
@@ -202,13 +202,13 @@ def delay(self, interval, loop=None):
         """ Add a time delay to results """
         return delay(interval, self, loop=None)
 
-    def combine_latest(self, *others):
+    def combine_latest(self, *others, **kwargs):
         """ Combine multiple streams together to a stream of tuples
 
         This will emit a new tuple of all of the most recent elements seen from
```

Comment:
add documentation, like in `rate_limit` something like :
```python
Parameters
---------------
emit_on : stream or list of streams or None
only emit from these streams listed. If None, emit from any stream

```
(note I avoid saying "parents" or "children" since  I think this could be interpreted two ways. parents are downstream in this code)